packages/sources/eth-beacon/src/transport/balance.ts

```diff
@@ -200,13 +199,14 @@ export class BalanceTransport extends SubscriptionTransport<BalanceTransportType
 
   private async queryBeaconChain(
     url: string,
-    addresses: string,
-    statusList?: string,
+    addresses: string[],
+    statusList?: string[],
   ): Promise<StateResponseSchema> {
```

Comment:
I guess the other option here if we don't want to require a specific version/ensure backward compatibility would be to add a feature flag somewhere here and toggle between GET/POST payloads based on that. I think we're OK with a strict cutover though as long as it's clear and documented somewhere (and GTM/opdata/DON Ops know to specify the RPC requirements).